subsys/bluetooth/controller/ll_sw/ctrl.c

```diff
@@ -52,22 +49,23 @@
 	(((octets) + 14) * 8 / BIT(((phy) & 0x03) >> 1))
 #endif /* !CONFIG_BT_CTLR_PHY_CODED */
 
-#define RADIO_TICKER_JITTER_US			16
-#define RADIO_TICKER_START_PART_US		300
-#define RADIO_TICKER_XTAL_OFFSET_US		1200
-#define RADIO_TICKER_PREEMPT_PART_US		0
-#define RADIO_TICKER_PREEMPT_PART_MIN_US	0
-#define RADIO_TICKER_PREEMPT_PART_MAX_US	RADIO_TICKER_XTAL_OFFSET_US
-
 #if defined(CONFIG_BT_CTLR_CONN_RSSI)
 #define RADIO_RSSI_SAMPLE_COUNT	10
 #define RADIO_RSSI_THRESHOLD	4
 #endif /* CONFIG_BT_CTLR_CONN_RSSI */
 
-#define RADIO_IRK_COUNT_MAX	8
-
 #define SILENT_CONNECTION	0
 
+#define RADIO_TIFS                      150
+#define RADIO_CONN_EVENTS(x, y)         ((u16_t)(((x) + (y) - 1) / (y)))
+
+#define RADIO_TICKER_JITTER_US           16
```

Comment:
Should these be part of the hal, or are they common to all radios?

Reply:
The value is used in a mixed up context, both in relation to BLE permitted sleep clock accuracy jitter of +/-16us and ticker's API limitation where in first interval having a jitter of half a 32KHz unit.
I will prefer to take this up at the next opportunity (true portability to other SoCs).

Reply:
OK, makes sense. I have no issues with the rest of the PR.